app/models/user.rb

```diff
@@ -3,9 +3,12 @@
 class User < ApplicationRecord
   has_secure_password
   has_secure_token :auth_token
+  has_secure_token :email_confirmation_token
```

Comment:
In the future we should start the token at `nil`. Out of scope for this PR, this note is to remind me to refactor this and `auth_token` later.